rbc/tests/__init__.py

```diff
@@ -257,6 +433,14 @@ def heavydb():
 
     f'{heavydb.table_name}timestamp' - contains timestamp t9, t6
                                   where 't' prefix is for timestamp.
+
+    f'{heavydb.table_name}geopoint' - contains columns with GeoPoint
+
+    f'{heavydb.table_name}linestring' - contains columns with LineString
+
+    f'{heavydb.table_name}multipoint' - contains columns with MultiPoint
+
+    f'{heavydb.table_name}polygon' - contains columns with Polygon
```

Comment:
Add also the corresponding doc lines for `multilinestring` and `multipolygon` for completeness.